docs/build-and-push-assets.md

```diff
@@ -81,7 +81,7 @@ jobs:
   build-assets:
     uses: inpsyde/reusable-workflows/.github/workflows/build-and-push-assets.yml@main
     with:
-      BUILT_BRANCH_SUFFIX: '-built' # Optionally, to push compiled assets to built branch
+      BUILT_BRANCH_NAME: "${{ github.ref_name }}-built" # Optionally, to push compiled assets to built branch
       RELEASE_BRANCH_NAME: 'release' # Optionally, to move tags to release branch
```

Comment:
Let's take the opportunity to remove the quotes (instead of introducing mixed double/single quotes).
```suggestion
      BUILT_BRANCH_NAME: ${{ github.ref_name }}-built # Optionally, to push compiled assets to built branch
      RELEASE_BRANCH_NAME: release # Optionally, to move tags to release branch
```
This should work according to https://www.yamllint.com, but please test this!